Gemfile.lock

```diff
@@ -15,77 +15,94 @@ GIT
 PATH
   remote: .
   specs:
-    ketcherails (0.1.12)
+    ketcherails (0.1.13)
       active_model_serializers (< 0.10.0)
       bootstrap-kaminari-views (= 0.0.5)
       bootstrap-sass
       daemons
       delayed_job_active_record
       grape (< 2.0)
       grape-active_model_serializers (~> 1.3.2)
+      grape-entity (~> 0.10.2)
       haml-rails
       httparty
       inchi-gem
       jquery-rails
       jquery-ui-rails (~> 5.0.5)
       kaminari
       mimemagic (>= 0.3.10)
```

Comment:
Now that paperclip is gone, you can remove the mimemagic dependency from the .gemspec file :)